js/signup.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(() => {
+  $("#signInBtn").click(() => {
+    $("#signUpDiv").hide();
+    $("#signInDiv").show().css("display", "flex");
+  });
+  $("#signUpBtn").click(() => {
```

Comment:
I explain again!
You didn't match with " signUpBtn" in HTML...

You have to add the match id " signUpBtn"  there.